recipes/c-ares/all/conanfile.py

```diff
@@ -72,6 +72,9 @@ def package_info(self):
             self.cpp_info.system_libs.extend(["ws2_32", "Advapi32"])
         elif self.settings.os == "Macos":
             self.cpp_info.system_libs.append("resolv")
+
+        self.cpp_info.names["cmake_find_package"] = "cares"
+        self.cpp_info.names["cmake_find_package_multi"] = "cares"
```

Comment:
```suggestion
        self.cpp_info.names["cmake_find_package"] = "CARES"
        self.cpp_info.names["cmake_find_package_multi"] = "CARES"
```

Your reference uses upper case

Reply:
No, the CMake variable names are always lowercase, but both references state the name actual lowercase

Reply:
Checking again, the c-ares project already provide target files:

```
-- Installing: c-ares/1.16.1/_/_/package/6af9cc7/lib/cmake/c-ares/c-ares-targets.cmake
-- Installing: c-ares/1.16.1/_/_/package/6af9cc7/lib/cmake/c-ares/c-ares-targets-release.cmake
-- Installing: c-ares/1.16.1/_/_/package/6af9cc7/lib/cmake/c-ares/c-ares-config.cmake
-- Installing: c-ares/1.16.1/_/_/package/6af9cc7/lib/cmake/c-ares/c-ares-config-version.cmake
-- Installing: c-ares/1.16.1/_/_/package/6af9cc7/lib/pkgconfig/libcares.pc
```

Thus, the name for cmake should be `c-ares` as we are using by default, however, for pkgconfig we should update to `cares`, just like your change.

Reply:
Why cares for pkgconfig? Why not libcares?

Reply:
@SpaceIm I didn't get, we are using libcares for pkgconfig, take a look at line 78